packages/cli/generators/relation/index.js

```diff
@@ -374,6 +380,25 @@ module.exports = class RelationGenerator extends ArtifactGenerator {
     });
   }
 
+  async promptRegisterInclusionResolver() {
+    // should we let users decide the relation is traversable?
+    // or promt a warning msg: 'Notice not all models are traversable'
+    return this.prompt([
+      {
+        type: 'confirm',
+        name: 'isInclusionResolverRegistered',
+        message: `Do you want to use inclusion for querying data over relation ${chalk.yellow(
```

Comment:
I am not sure if this message makes sense. Any suggestions to make it more intuitive?

And also, do we want to prompt a warning message that not all models are traversable?

Reply:
> Any suggestions to make it more intuitive?

+1 in making it more intuitive. Inspired by the description in https://github.com/strongloop/loopback-next/issues/1352#issuecomment-433300937, I'm thinking something along the line:

```
Do you want to traverse the data in models defined in relation {relation_name}?
```

WDYT?

> And also, do we want to prompt a warning message that not all models are traversable?

If we're going to have a warning message (I'm good either way), maybe we can use something like https://github.com/strongloop/loopback-next/blob/master/packages/cli/generators/interceptor/index.js#L124-L127?

Reply:
Because not all relations are allowed to be traversed,  registering such resolver might be problematic/unworkable. 
We documented it in [our site](https://loopback.io/doc/en/lb4/HasMany-relation.html#querying-related-models) already. I don't know if we usually warn users again in CLI or just prompt the question.

Reply:
Few prompts to consider:

- `Allow ${sourceModelName} queries to include data from related {targetModelName} instances?`
- `Allow ${sourceModelName} queries to include data from {targetModelName} instances related via ${relationName}?`
- `Allow queries to traverse from {sourceModelName} to {targetModelName} via {relationName}?`

I think the first prompt will make it pretty obvious to users if they are introducing unwanted traversals:
> Allow User queries to include data from related AccessToken instances?